package.json

```diff
@@ -1,13 +1,6 @@
 {
-  "name": "settings-sharelatex",
-  "homepage": "www.sharelatex.com",
-  "description": "A centralised settings system for ShareLaTeX",
-  "version": "1.3.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/sharelatex/settings-sharelatex.git"
-  },
-  "dependencies": {
-    "coffee-script": "1.6.0"
-  }
+  "name": "@overleaf/settings-module",
```

Comment:
We are omitting the `-module` suffix in our other modules in the `@overleaf` namespace. WDYT about omitting it here as well? https://www.npmjs.com/search?q=%40overleaf

Reply:
I thought about that, but `@overleaf/settings` maybe implies that it's the actual settings?

I don't mind much either way, but this repo should probably also be renamed if the name is going to change.

Reply:
> I thought about that, but `@overleaf/settings` maybe implies that it's the actual settings?

Upon importing it you do get the actual config of the current service ;)

Reply:
That's true 😄

Reply:
Yeah, naming is hard. We've got
- `redis-wrapper` -> `@overleaf/redis-wrapper`
-  `metrics-module` -> `@overleaf/metrics`
-  `o-error` -> `@overleaf/o-error`

WDYT about a poll in slack what others think?

Reply:
Either a poll or I'm happy for @jdleesmiller to weigh in. The precedent of `metrics-module` -> `@overleaf/metrics` implies that `settings-module` -> `@overleaf/settings` would be fine, so maybe let's go for that?

Reply:
> let's go for that?

OK by me! I am persuaded by https://github.com/overleaf/settings-module/pull/5#discussion_r640513093 :)